.prettierignore

```diff
@@ -5,4 +5,5 @@ dist
 pnpm-lock.yaml
 .vercel
 _generated.md
-land-110m.json
\ No newline at end of file
+land-110m.json
+/apps/svelte.dev/content/docs
```

Comment:
We should be able to revert this too. I'm assuming this was done because Prettier had other opinions about what you were trying to check in?

Reply:
it did. i think it'll save us bother in future, so i figured we may as well keep it